server/routerlicious/kubernetes/routerlicious/values.yaml

```diff
@@ -54,7 +54,7 @@ mongodb:
   url: mongodb_url
 
 rabbitmq:
-  connectionString: "connection_string"
+  connectionString: ""
```

Comment:
For my own understanding, why do we make the change here and not in server/routerlicious/packages/routerlicious/config/config.json ?

Reply:
Cause we want to keep running rabbitmq locally. This file is what actually goes to cluster. There is another step of overriding this value but thats a deployment level change.